index.d.ts

```diff
@@ -203,8 +203,8 @@ declare module 'binance-api-node' {
   }
 
   export interface SetBNBBurnOptions {
-    spotBNBBurn: boolean
-    interestBNBBurn: boolean
+    spotBNBBurn?: boolean | booleanString
+    interestBNBBurn?: boolean | booleanString
```

Comment:
what's booleanString?

Reply:
it's a new type from previous pull request

Reply:
I don't think it's the best decision but Binance API has a lot of mess about boolean things

Reply:
it was created here https://github.com/Ashlar/binance-api-node/pull/493

Reply:
sounds good